spec/status.xml

```diff
@@ -5,62 +5,47 @@
+and to encourage implementation by the developer community. This Candidate Recommendation is expected to 
+advance to Proposed Recommendation no earlier than &crEnd;.</p>
 
-<p>This version reflects changes made as a result of applying errata and addressing
-comments received since the Recommendation was first published. With one minor
-exception noted below, these changes are editorial.</p>
+<p>This new edition of the Timed Text Markup Language (TTML) 1.0
+<bibref ref="ttml10"/> clarifies ambiguities and corrects errors identified in the 
+<loc href="&prevrec;">previous
+edition</loc>, which this new edition supersedes. No features were added.</p>
 
-<p>This document is a revision of Timed Text Markup Language (TTML) 1.0
-<bibref ref="ttml10"/>, superseding that <loc href="http://www.w3.org/TR/2010/REC-ttaf1-dfxp-20101118/">previous
-Recommendation</loc>. The only identified substantive difference between this version and
-its predecessor is a change of the default treatment of the <att>ttp:markerMode</att> parameter attribute
-from <code>continuous</code> to <code>discontinuous</code>. A cumulative summary of all changes applied
-to this version since the Recommendation was first published is available at
-<loc  href="ttml1-changes.html#change-history-rec-to-2e-per">Changes
-from Recommendation to Proposed Edited Recommendation (2nd Ed.)</loc> and
-<loc  href="ttml1-changes.html#change-history-2e-per-to-rec-2e">Changes
-from Proposed Edited Recommendation (2nd Ed.) to Recommendation (2nd Ed.)</loc>.</p>
+<p>Substantive changes applied since the <loc href="&prevloc;">previous version of this specification</loc> are listed at 
+<loc href="substantive-changes-summary.txt">substantive-changes-summary.txt</loc>.</p>
 
-<p>A <loc href="http://www.w3.org/2008/10/dfxp-testsuite.zip">sample test suite</loc> is available. This set of tests is not intended to
-be exhaustive. The <loc href="http://www.w3.org/2009/05/dfxp-results.html">implementation report</loc> used by the director to
-transition to Recommendation has been made available. There have been no formal objections
-to the publication of this document.</p>
+<p>For convenience, a diff between this and the <loc href="&prevrec;">previous edition</loc> 
+is offered at the <loc href="https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2013%2FREC-ttml1-20130924%2F%26doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2018%2FCR-ttml1-20180313%2F">W3C HTML Diff service</loc>.</p>
 
-<p>This document was produced by the <loc
-href="http://www.w3.org/AudioVideo/TT/">Timed Text (TT) Working Group</loc> as
-part of the W3C <loc href="http://www.w3.org/2008/WebVideo/">Video
-in the Web Activity</loc>, following the procedures set out for the W3C <loc
-href="http://www.w3.org/Consortium/Process/">Process</loc>.  The authors of this
-document are listed in the header of this document, including input and review
```

Comment:
I'm still uncomfortable removing the sentence describing the contributors including EBU and SMPTE, in case it is noticed and considered to be a negative comment in the direction of EBU and SMPTE.

Reply:
They are listed in more details in the acknowledgement section.

Reply:
I don't see them listed in more detail there, maybe in _less_ detail. Why is the SOTD not the place?

Reply:
Checking back, this sentence was not present in the first TTML1 Rec, so there's precedent for omitting it. In the case of this 3rd Edition, I'm not sure if there are any specific external organisations who gave review to include yet. Possibly we could reinstate a similar sentence when progressing to PR or Rec based on any wide review input received at that time.